db/data/sample/groups.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
This sample data, not just this file, should be removed. The method in which demo data is installed should be automatic.

The data and task need to be refactored and cannot be kept as is.